src/scripts/main.js

```diff
@@ -1,7 +1,75 @@
-'use strict';
+const Game = require('../modules/Game.class');
+
+// eslint-disable-next-line no-unused-expressions
+('use strict');
```

Comment:
The 'use strict' directive is usually placed at the beginning of a script, it's not working after the first executable statement.